types/argparse/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for argparse 1.0
+// Type definitions for argparse 1.1
```

Comment:
There is no 1.1: https://www.npmjs.com/package/argparse

Did you mean 1.0.1?

Reply:
I wanted to increase the version of the type definition, because the modification to `ArgumentOptions` is technically backwards incompatible.

But reading the [DefinitelyTyped versioning documentation](https://github.com/DefinitelyTyped/DefinitelyTyped/#how-do-definitelytyped-package-versions-relate-to-versions-of-the-corresponding-library) more closely, it appears that there is no way to do what I want.  I'll change it back to "1.0".

Reply:
Right. It should simply stay that way because `@types/` match major.minor  of their corresponding library package.